README.md

```diff
@@ -65,6 +65,29 @@ The app automatically launches with the Chromium developer tools opened by defau
 
 Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso. See [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) for details on setting up your editor.
 
+### Testing
+
+#### Running tests
+
+You can run tests with the following command:
+
+```bash
+npm run test
```

Comment:
Nitpick: The command `npm test` also runs the unit tests.

Reply:
It may also be nice to note that individual tests can be run using, for example, `npm test src/components/tests/add-site.test.tsx`, similar to Carlos' comment about [the E2E docs](https://github.com/Automattic/studio/pull/190/files#r1620223737).